.gitignore

```diff
@@ -39,9 +39,6 @@ Vagrantfile
 /config/aws.yml
 /keys/*.key.enc
 !/keys/*.key.enc.example
-/keys/equifax_rsa
```

Comment:
How about we leaves these in so the file people have locally won't start getting tracked?